src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -71,22 +73,15 @@ internal sealed class EslintBridgeAnalyzer : IEslintBridgeAnalyzer
 
         public async Task<IReadOnlyCollection<IAnalysisIssue>> Analyze(string filePath, string tsConfig, CancellationToken cancellationToken)
         {
-            try
-            {
-                return await GetFileIssues(filePath, tsConfig, cancellationToken);
-            }
-            catch (EslintBridgeClientNotInitializedException)
+            await EnsureEslintBridgeClientIsInitialized(cancellationToken);
+            var analysisResponse = await eslintBridgeClient.Analyze(filePath, tsConfig, cancellationToken);
+
+            if (LinterNotInitializedResponse(analysisResponse))
```

Comment:
I still don't think it's obvious _why_ we might get a "not initialised" error having just called `EnsureEslintBridgeClientIsInitialized`. I think it needs a comment e.g.

```
// The call to `EnsureEslintBridgeClientIsInitialized` above doesn't guarantee the client is correctly initialized (e.g. the external process might have crashed).
// So we still need to handle the "not initialised" case here.
```